src/hssm/plotting/model_cartoon.py

```diff
@@ -0,0 +1,1985 @@
+            ].shape[0],
+        )
+
+        # Add histograms for posterior mean simulation
+        axis_twin_up.hist(
+            np.abs(
+                sim_out["rts"][(sim_out["rts"] != -999) & (sim_out["choices"] == 1)]
+            ),
+            bins=bins,
+            weights=weights_up,
+            histtype=hist_histtype,
+            bottom=hist_bottom_high,
+            alpha=1,
+            color=color_pp_mean,
+            edgecolor=color_pp_mean,
+            linewidth=linewidth_histogram,
+            zorder=-1,
+        )
+
+        axis_twin_down.hist(
+            np.abs(
+                sim_out["rts"][(sim_out["rts"] != -999) & (sim_out["choices"] != 1)]
+            ),
+            bins=bins,
+            weights=weights_down,
+            histtype=hist_histtype,
+            bottom=hist_bottom_low,
+            alpha=1,
+            color=color_pp_mean,
+            edgecolor=color_pp_mean,
+            linewidth=linewidth_histogram,
+            zorder=-1,
+        )
+
+    if theta_posterior is not None:
+        # Add histograms for posterior samples:
+        for k, sim_out_tmp in posterior_pred_sims.items():
+            weights_up = np.tile(
+                (1 / bin_size)
+                / sim_out_tmp["rts"][(sim_out_tmp["rts"] != -999)].shape[0],
+                reps=sim_out_tmp["rts"][
+                    (sim_out_tmp["rts"] != -999) & (sim_out_tmp["choices"] == 1)
+                ].shape[0],
+            )
+            weights_down = np.tile(
+                (1 / bin_size)
+                / sim_out_tmp["rts"][(sim_out_tmp["rts"] != -999)].shape[0],
+                reps=sim_out_tmp["rts"][
+                    (sim_out_tmp["rts"] != -999) & (sim_out_tmp["choices"] != 1)
+                ].shape[0],
+            )
+
+            # Add histograms for posterior samples
+            axis_twin_up.hist(
+                np.abs(
+                    sim_out_tmp["rts"][
+                        (sim_out_tmp["rts"] != -999) & (sim_out_tmp["choices"] == 1)
+                    ]
+                ),
+                bins=bins,
+                weights=weights_up,
+                histtype=hist_histtype,
+                bottom=hist_bottom_high,
+                alpha=alpha_pp,
+                color=color_pp,
+                edgecolor=color_pp,
+                linewidth=linewidth_histogram,
+                zorder=(-k - 1),
+            )
+
+            axis_twin_down.hist(
+                np.abs(
+                    sim_out_tmp["rts"][
+                        (sim_out_tmp["rts"] != -999) & (sim_out_tmp["choices"] != 1)
+                    ]
+                ),
+                bins=bins,
+                weights=weights_down,
+                histtype=hist_histtype,
+                bottom=hist_bottom_low,
+                alpha=alpha_pp,
+                color=color_pp,
+                edgecolor=color_pp,
+                linewidth=linewidth_histogram,
+                zorder=(-k - 1),
+            )
```

Comment:
This is almost the same logic as above. Consider moving this to a helper function and reuse it.